hypervisor/include/lib/bits.h

```diff
@@ -169,6 +172,22 @@ static inline uint16_t clz64(uint64_t value)
 	}
 }
 
+static inline uint64_t ffz64_ex(const uint64_t *addr, uint64_t size)
+{
+	uint64_t idx;
+
+	if ((size & (BITS_PER_LONG - 1UL)) != 0UL) {
```

Comment:
how about use (8* sizeof(*addr)) to replace BITS_PER_LONG?  Here it is not LONG, it is uint64_t.